UPGRADE.md

```diff
@@ -1,15 +1,40 @@
-# Upgrade
+Upgrade
+=======
```

Comment:
I would prefer the other variant of headlines, since we do it like that in all the markdown files, and it is easier to change the headline.